src/OsLogin/README.md

```diff
@@ -0,0 +1,18 @@
+# Google Cloud OsLogin for PHP
```

Comment:
Could you add the badges and gRPC notice? [Here's a template.](https://raw.githubusercontent.com/GoogleCloudPlatform/google-cloud-php/master/src/Bigtable/README.md)

I updated the component CLI to add both of these, so pull latest to get it for next time. :)

Reply:
Done